README.md

```diff
@@ -28,102 +28,103 @@ The CI environment supports both ARM and Bicep and can be leveraged using GitHub
 
 | Name | Status |
```

Comment:
Undo updates. This is done automatically by the CI environment.

Reply:
Copy/paste upstream file